Modules/MUON/MCH/test/testMuonChambers.cxx

```diff
@@ -0,0 +1,25 @@
+///
+/// \file   testMuonChambers.cxx
+/// \author
+///
+
+#include "QualityControl/TaskFactory.h"
+
+#define BOOST_TEST_MODULE Publisher test
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+
+#include <boost/test/unit_test.hpp>
+
+namespace o2
+{
+namespace quality_control_modules
+{
+namespace muonchambers
+{
+
+BOOST_AUTO_TEST_CASE(instantiate_task) { BOOST_CHECK(true); }
```

Comment:
you might want to test your decoding at some point

Reply:
We are setting up the tests at the O2 level...

Reply:
excellent !